bindings/python/iota_sdk/types/output_with_extended_metadata.py

```diff
@@ -12,20 +12,20 @@
 
 @json
 @dataclass
-class OutputData:
-    """Output data.
+class OutputWithExtendedMetadata:
+    """Output with extended metadata.
 
     Attributes:
-        output_id: With the output data corresponding output ID.
-        metadata: With the output corresponding metadata.
-        output: The output object itself.
+        output: The output itself.
+        metadata: The metadata of the output.
         output_id_proof: The output ID proof.
+        output_id: The corresponding output ID.
         network_id: The network ID the output belongs to.
         remainder: Whether the output represents a remainder amount.
     """
-    output_id: OutputId
-    metadata: OutputMetadata
     output: Output
+    metadata: OutputMetadata
     output_id_proof: OutputIdProof
+    output_id: OutputId
```

Comment:
isn't the ID already in the metadata?

Reply:
yes it is, and I mentioned this weirdness already, but should probable be dealt with separately as this would require some discussion. AFAIR this is done on purpose, because the metadata can't be trusted 100%.